pkg/operator/monitoring/metrics.go

```diff
@@ -0,0 +1,67 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package monitoring
+
+import (
+	"context"
+
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+)
+
+var (
+	serviceCreatedTotal = stats.Int64(
+		"operator/service_created_total",
+		"The total number of dapr service created.",
+		stats.UnitDimensionless)
+	serviceDeletedTotal = stats.Int64(
+		"operator/service_deleted_total",
+		"The total number of dapr service deleted.",
+		stats.UnitDimensionless)
+	serviceUpdatedTotal = stats.Int64(
+		"operator/service_updated_total",
+		"The total number of dapr service updated.",
+		stats.UnitDimensionless)
+
+	noKeys = []tag.Key{}
+)
+
```

Comment:
Can we add `app_id` tag to each metric? (tag term in oc is same as metric dimension)

Reply:
done . please check.

Reply:
done. please check.